src/Compilers/Core/Portable/WellKnownMember.cs

```diff
@@ -515,6 +515,8 @@ internal enum WellKnownMember
 
         System_Runtime_CompilerServices_DefaultInterpolatedStringHandler__ToStringAndClear,
 
+        System_Index__op_Implicit_FromInt32,
```

Comment:
Is this change caused by a spec somewhere?

Reply:
> Is this change caused by a spec somewhere?

Tacked by [https://github.com/dotnet/roslyn/issues/57825](https://github.com/dotnet/roslyn/issues/57825)